modules/packages/Vector.chpl

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Can likely use much more minimal push-back vector implementation (no we're not using `list`)

Reply:
I really think we can just discard this entire module and go a more minimal approach.